lib/questions.rb

```diff
@@ -1,61 +1,84 @@
 # keep only the elements that start with an a
 def select_elements_starting_with_a(array)
+  array.select{|element| element[0] == "a"}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.